app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to action: @user
```

Comment:
It is better to use `new_user_path`:
`redirect_to new_user_path` if user was saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))